src/test/java/com/exasol/dbbuilder/dialects/mysql/MySqlImmediateDatabaseObjectWriterTest.java

```diff
@@ -0,0 +1,46 @@
+package com.exasol.dbbuilder.dialects.mysql;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.instanceOf;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.when;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.sql.Connection;
+import java.sql.SQLException;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import com.exasol.dbbuilder.dialects.DatabaseObject;
+import com.exasol.dbbuilder.dialects.DatabaseObjectException;
+
+@ExtendWith(MockitoExtension.class)
+class MySqlImmediateDatabaseObjectWriterTest {
```

Comment:
This test also looks completely dialect unspecific.

Reply:
fixed